resources/views/artists/index.blade.php

```diff
@@ -0,0 +1,52 @@
+{{--
+    Copyright 2016 ppy Pty. Ltd.
+
+    This file is part of osu!web. osu!web is distributed with the hope of
+    attracting more community contributions to the core ecosystem of osu!.
+
+    osu!web is free software: you can redistribute it and/or modify
+    it under the terms of the Affero GNU General Public License version 3
+    as published by the Free Software Foundation.
+
+    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+--}}
+@extends("master", [
+    'current_section' => 'beatmaps',
+    'current_action' => 'artists',
+    'title' => "Featured Artists",
+    'pageDescription' => "Something, something, artists!",
+    'body_additional_classes' => 'osu-layout--body-darker'
+])
+
+@section("content")
+    <div class="osu-layout__row">
+        <div class="osu-page-header-v2 osu-page-header-v2--featured-artists">
+            <div class="osu-page-header-v2__overlay"></div>
+            <div class="osu-page-header-v2__subtitle osu-page-header-v2__subtitle--artist">Beatmaps &raquo;</div>
+            <div class="osu-page-header-v2__title">Featured Artists</div>
+        </div>
+    </div>
+    <div class="osu-layout__row osu-layout__row--page-artist">
+        <div class="page-contents page-contents--artist">
+            <div class="page-contents__content--artist-left">
+                <div class="artist__index">
+                    @foreach ($artists as $artist)
+                        <div class="artist__box">
+                            <a href="{{route('artist.show', ['id' => $artist->id])}}" class="artist__portrait artist__portrait--index" style="{{$artist->cover_url ? 'background-image: url(' . $artist->cover_url . ')' : ''}}"></a>
+                            @if($artist->label !== null)
+                                <a class="artist__label-overlay artist__label-overlay--index" href="{{$artist->label->website}}" style="background-image: url('{{$artist->label ? $artist->label->icon_url : ''}}')"></a>
```

Comment:
...oh I see ಠ_ಠ

Better use `top` instead of bottom in this case since it's easier to measure (`portrait height - label height`) instead of the height of two text lines (`font size 1 * line height 1 + margin 1 + font size 2 * line height 2 + margin 2`)?